aeron-samples/src/main/java/io/aeron/samples/mdc/AeronClientPing.java

```diff
@@ -0,0 +1,205 @@
+        private static final String address = "localhost";
+        private static final int port = 13000;
+        private static final int controlPort = 13001;
+        private static final ChannelUriStringBuilder outboundChannelBuilder =
+            new ChannelUriStringBuilder()
+                .endpoint(address + ':' + port)
+                .reliable(Boolean.TRUE)
+                .media("udp");
+        private static final ChannelUriStringBuilder inboundChannelBuilder =
+            new ChannelUriStringBuilder()
+                .controlEndpoint(address + ':' + controlPort)
+                .controlMode(CommonContext.MDC_CONTROL_MODE_DYNAMIC)
+                .reliable(Boolean.TRUE)
+                .media("udp");
+
+        private final Histogram histogram = new Histogram(TimeUnit.SECONDS.toNanos(10), 3);
+        private final MsgFragmentHandler fragmentHandler = new MsgFragmentHandler(histogram);
+        private final IdleStrategy idleStrategy = new BackoffIdleStrategy(1, 1, 1, 100);
+
+        private final Aeron aeron;
+
+        private Client(Aeron aeron)
+        {
+            this.aeron = aeron;
+        }
+
+        private void start()
+        {
+            int sid = new Random().nextInt(Integer.MAX_VALUE);
+            String outboundChannel = outboundChannelBuilder.sessionId(sid).build();
+            Publication publication = aeron.addExclusivePublication(outboundChannel, STREAM_ID);
+
+            int sessionId = publication.sessionId();
+
+            String inboundChannel =
+                inboundChannelBuilder.sessionId(sessionId ^ Integer.MAX_VALUE).build();
+
+            Subscription subscription =
+                aeron.addSubscription(
+                    inboundChannel,
+                    STREAM_ID,
+                    image ->
+                        System.out.println(
+                            "onClientImageAvailable: "
+                                + image.sessionId()
+                                + " / "
+                                + image.sourceIdentity()),
+                    image ->
+                        System.out.println(
+                            "onClientImageUnavailable: "
+                                + image.sessionId()
+                                + " / "
+                                + image.sourceIdentity()));
+
+            MsgPublication msgPublication = new MsgPublication(sessionId, publication);
+
+            // wait for available image
+            while (true)
+            {
+                if (subscription.imageCount() > 0)
+                {
+                    break;
+                }
+                idleStrategy.idle();
+            }
+
+            idleStrategy.reset();
+            histogram.reset();
+
+            Image image = subscription.images().get(0);
+
+            for (int i = 0; i < NUMBER_OF_MESSAGES; )
+            {
+
+                ExpandableArrayBuffer buffer = new ExpandableArrayBuffer(Long.BYTES);
```

Comment:
Ok, but I didn't find a DirectBuffer allocator which I can use to allocate buffers inside it and reuse them to reduce allocate memory. 
I'd like to send buffers and receive them asynchronously even though it will in one thread of eventLoop. 
Ok, maybe I need to create only one buffer and reuse it. But I don't know about its potential size.
I used to have netty-buffers and I can allocate and release them easily.